src/modules/Auth/api/api.ts

```diff
@@ -13,6 +13,8 @@ export const signInApi = ({ email, password }: SignIn, signal?: AbortSignal) =>
 
 export const logOutApi = () => authApiClientWithoutRefresh.post('auth/logout');
```

Comment:
just for convenience and to make the code more understandable, could you please name deleteAccessTokenApi and deleteRefreshTokenApi?

Reply:
done